frontend/src/styles/components.css

```
@@ -0,0 +1,296 @@
+   Shared component styles for Odin frontend.
+   Ported from odin/static/css/index/base.css,
+   odin/static/css/index/sensors.css, and
+   odin/static/css/modal.css with BEM naming.
+   ============================================================ */
+
+/* ----- Grid (responsive columns defined in responsive.css) ----- */
+.grid {
+  display: grid;
+  width: 100%;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-column-gap: 1.5rem;
```

Stylelint (17.14.0)

[error] Expected "grid-column-gap" to be "column-gap" (property-no-deprecated)

(property-no-deprecated)